fda-model/structures/Inventory.cs

```diff
@@ -181,9 +181,8 @@ private void LoadStructuresFromSourceFiles()
 
                 int fid = GetRowValueForColumn<int>(row, _map.StructureIDCol, defaultMissingValue);               
                 double val_struct = GetRowValueForColumn<double>(row,_map.StructureValueCol, defaultMissingValue);
-
-                string st_damcat = GetRowValueForColumn(row, _map.DamageCatagory, "NA");
                 string occtype = GetRowValueForColumn(row, _map.OccTypeCol, "NA");
+                string st_damcat = _occtypes[occtype].DamageCategory;
```

Comment:
This should probably always work, but if we think there is any chance that the occtype gets set to "NA" then we should check that the key exists first. 
![image](https://user-images.githubusercontent.com/47757848/212425767-495267c7-d254-40c6-9951-982ef11879d3.png)

Reply:
done